src/api/app/jobs/update_released_binaries_job.rb

```diff
@@ -9,4 +9,9 @@ def perform(event_id)
 
     BinaryRelease.update_binary_releases(repo, pl['payload'], event.created_at)
   end
+
+  def max_run_time
+    # twice the time of the default (8 hours now)
+    28800
```

Comment:
This won't work by itself as I have written in my previous comment. The delayed_job README explains why:

> NOTE: this can ONLY be used to set a max_run_time that is lower than Delayed::Worker.max_run_time. Otherwise the lock on the job would expire and another worker would start the working on the in progress job.

If we use this approach, we need to increase the timeout of all jobs while lowering the timeout of all jobs which don't require that much time. A better approach is the second option I proposed in my previous comment:

> (...) apply what is proposed [here](https://stackoverflow.com/questions/26179508/timeout-in-a-delayed-job). This would only be in the jobs which really need more time, the remaining jobs are unchanged.